planning/behavior_path_planner/src/behavior_path_planner_node.cpp

```diff
@@ -648,7 +650,19 @@ void BehaviorPathPlannerNode::run()
   }
 
   // for debug
-  debug_avoidance_msg_array_publisher_->publish(bt_manager_->getAvoidanceDebugMsgArray());
+  {
```

Comment:
@zulfaqar-azmi-t4 please move these code to a separated function. I want to make the main function shorter.

Reply:
Done. Fixed in 57c1ed16af196a0dfcd745d11a0ce6065fccac54